TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmode/teleop/TeleOpAlt.java

```diff
@@ -0,0 +1,70 @@
+package org.firstinspires.ftc.teamcode.opmode.teleop;
```

Comment:
I like what you are doing with the event-driven stuff. It does bring up the question of how we are going to implement something if we need to run a loop (ie position tracking, etc) which won't be event-driven.

Reply:
Yeah, as we continue I think maintaining and adjusting TeleOpAlt is going to be hard. I've been debating whether to rewrite it, delete it, or not. Most likely I'll remove it and clean up TeleOpMain to be more readable.

Reply:
If you take a look at https://github.com/BotsBurgh/BOTSBURGH-FTC-2021-22/blob/main/FtcRobotController/src/main/java/org/firstinspires/ftc/robotcontroller/external/samples/BasicOpMode_Iterative.java, you can get some ideas on how you can implement something sort of event driven that won't have these issues. I've been meaning to switch but I never got around to it.